src/sidecar/checkArchitecture.test.ts

```diff
@@ -11,7 +11,7 @@ const platformArchToBinaryName: Map<string, string> = new Map([
   ["linux-arm64", join(binariesDir, "linux-arm64")],
   ["darwin-x64", join(binariesDir, "osx-amd64")],
   ["darwin-arm64", join(binariesDir, "osx-aarch64")],
-  ["windows-x64", join(binariesDir, "windows-x64")],
```

Comment:
`Should throw error with incorrect binary` was failing because we were trying to look up `win32-x64` here and returning `undefined`